sqlx-core/src/postgres/options/mod.rs

```diff
@@ -319,6 +322,34 @@ impl PgConnectOptions {
         self
     }
 
+    /// Sets the extra options. Defaults to None
+    ///
+    /// # Example
+    ///
+    /// ```rust
+    /// # use sqlx_core::postgres::PgConnectOptions;
+    /// let options = PgConnectOptions::new()
+    ///     .options([("geqo", "off"), ("statement_timeout", "5min")]);
+    /// ```
+    pub fn options<K, V, I>(mut self, options: I) -> Self
+    where
+        K: ToString,
+        V: ToString,
```

Comment:
```suggestion
        K: Display,
        V: Display,
```

Reply:
I know I originally specified `ToString`, but `Display` is more optimal with this implementation.